python/paddle/fluid/layers/nn.py

```diff
@@ -1012,7 +1012,10 @@ def cross_entropy(input, label, soft_label=False):
                                tensor<float/double> with shape [N x D].
         soft_label (bool): a flag indicating whether to
                                            interpretate the given labels as soft
-                                           labels, default `False`.
+                                           labels. Default: `False`.
+        ignore_index (int): Specifies a target value that is ignored and does 
```

Comment:
Is there other way to mask label with some op? so gradient doesn't get propagated? I don't like this way of solution. What if you need to ignore multiple index in the future?